src/classes/UTIL_Typeahead_TEST.cls

```diff
@@ -49,11 +49,11 @@ public with sharing class UTIL_Typeahead_TEST {
 	      Test.setFixedSearchResults(fixedSearchResults);
 	
 	    Test.startTest();
-	    list<sObject> accts = UTIL_Typeahead.searchRecords( 'test', 'Account', null, null, null, null, null );
+	    list<sObject> accts = UTIL_Typeahead.searchRecords( 'test', 'Account', null, null, null, 'Id', null );
```

Comment:
Is there a change needed in the `UTIL_Typeahead.component` file as well? I noticed that this method is called from that component passing in a value for the sort parameter, which could be null (or `Name` - the default value in the component).